source/ada/lsp-ada_handlers-named_parameters_commands.adb

```diff
@@ -217,23 +222,51 @@ package body LSP.Ada_Handlers.Named_Parameters_Commands is
                Params : constant Libadalang.Analysis.Param_Spec_Array :=
                  Spec.P_Params;
             begin
-               Append (Params);
+               return Params;
             end;
 
          when Libadalang.Common.Ada_Basic_Subp_Decl =>
             declare
-               Spec : constant Libadalang.Analysis.Base_Subp_Spec :=
+               Spec   : constant Libadalang.Analysis.Base_Subp_Spec :=
                  Decl.As_Basic_Subp_Decl.P_Subp_Decl_Spec;
                Params : constant Libadalang.Analysis.Param_Spec_Array :=
```

Comment:
The Params variables are only used once and can be removed

Reply:
There is no rule that says "You should only create variables when it's used multiple times": variables improve readability and allow easier further refactorings, so we should keep it IMO.

Reply:
This is not necessary, just my preference, this is why the PR was also approved